packages/runtime/container-runtime/src/inboundBatchAggregator.ts

```diff
@@ -324,5 +222,40 @@ class ScheduleManagerCore {
 			// Continuation of current batch. Do nothing
 			assert(this.currentBatchClientId !== undefined, 0x2a1 /* "logic error" */);
 		}
+	};
+
+	private pauseQueue() {
```

Comment:
Why move this stuff? Smaller diffs are nice, but feel free to leave it if you like the organization better this way

Reply:
I changed the organization to keep the event handlers together and move these two together which are called by both the event handlers.